scripts/test_cargo_omnibump_contract.py

```
@@ -0,0 +1,276 @@
+
+def skip(reason: str) -> None:
+    """Local runs may lack docker or a registry; CI must prove the contract."""
+    message = f"scripts/test_cargo_omnibump_contract.py: {reason}"
+    if os.environ.get("CI"):
+        raise SystemExit(f"error: {message}")
+    print(f"SKIPPED {message}")
+    raise SystemExit(0)
+
+
+def image(work: Path) -> str:
+    if subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)